local-tests/chainrunner/node.py

```diff
@@ -0,0 +1,92 @@
+import json
+import os.path as op
+import re
+import requests
+import jsonrpcclient as rpc
+import subprocess
+
+from .utils import flags_from_dict
+
+class node:
+    """A class representing a single node of a running blockchain."""
+    def __init__(self, binary, chainspec, path, logdir=None):
```

Comment:
It is highly not obvious what `path` refers to

Reply:
Added some docstrings